src/main/java/madstax/view/PlannerCellRenderer.java

```diff
@@ -4,17 +4,23 @@
 import org.apache.commons.lang3.StringUtils;
 
 import javax.swing.*;
+import javax.swing.border.MatteBorder;
 import javax.swing.table.DefaultTableCellRenderer;
 import javax.swing.table.TableCellRenderer;
 import java.awt.*;
 import java.util.ArrayList;
 
 @SuppressWarnings("unchecked")
 public class PlannerCellRenderer extends JLabel implements TableCellRenderer {
+    private int selectedRow;
```

Comment:
The table has its own API to tell you which row is currently selected.

Reply:
I fixed this - it helped with the issue of selecting 2 rows that you mentioned as well!